.github/workflows/issues.yml

```diff
@@ -24,7 +24,7 @@ jobs:
               owner: context.repo.owner,
               repo: context.repo.repo
             });
-            return issue.data.assignee.login || '';
```

Comment:
assignee is coming back as 'null' now for some reason. Github may have updated their API.